### ISSUE TITLE
Ensure that the correct rgroup files are run for the correct GCM-RCP-time period combination.

### ISSUE THREAD

**Issue by chaukap** [DrylandEcology/rSFSTEP2#115]
Currently the code will run every rgroup file for every RCP-GCM-time period combination. We need to ensure that, if space is readjusted, we only run readujsted space rgroup files for the corresponding climate.

**Comment by chaukap** [comment]
@kpalmqui With commit 9a97a84 the code will now generate the correct files with readjusted space parameters and run them for the correct GCM-RCP-time period combination. I think I am ready for you to review the changes and see if there is anything that needs tweaking.

**Comment by kpalmqui** [comment]
@chaukap 

Yes, the correct number of files are generated. Can you potentially name them according to the GCM-RCP-time-period combination they were generated with, instead of 1,2,3... etc.? 

It may be hard for the user to figure out which corresponds to which. For instance I assume "1" = "Current". But what about 2 and 3? Is that RCP4.5.GCM1.50 and RCP4.5.GCM1.90? Or RCP4.5.GCM1.50 and RCP4.5.GCM2.50?
